src/settings/ManageAuthorityFiles/ManageAuthorityFiles.js

```diff
@@ -0,0 +1,121 @@
+import { useIntl } from 'react-intl';
+
+import {
+  useStripes,
+  TitleManager,
+  checkIfUserInMemberTenant,
+} from '@folio/stripes/core';
+import { ControlledVocab } from '@folio/stripes/smart-components';
+import {
+  InfoPopover,
+  Label,
+  Loading,
+} from '@folio/stripes/components';
+import { useUserTenantPermissions } from '@folio/stripes-authority-components';
+
+import { getAuthorityFormatter } from './getAuthorityFormatter';
+import { getFieldComponents } from './getFieldComponents';
+import {
+  authorityFilesColumns,
+  SOURCES,
+} from './constants';
+
+const ManageAuthorityFiles = () => {
+  const stripes = useStripes();
+  const { formatMessage } = useIntl();
+  const ConnectedControlledVocab = stripes.connect(ControlledVocab);
+
+  const userId = stripes?.user?.user?.id;
+  const centralTenantId = stripes.user.user?.consortium?.centralTenantId;
```

Comment:
You have optional `stripes` for `userId` and required one for `centralTenantId`, choose 1 of them, please